hw/ip/spi_device/doc/_index.md

```diff
@@ -246,6 +276,36 @@ All reads and writes to/from the SRAM for RXF and TXF activity are managed by
 direct reads and writes through the TLUL bus interface, managed by the
 auto-generated register file control logic.
 
+## TPM over SPI
+
+![TPM over SPI block diagram](tpm-blockdiagram.svg)
+
+The TPM over SPI submodule processes the fundamental data only.
```

Comment:
does this mean it doesn't do any of the high level features? I feel like `low level data` only might be clearer.
what do you think?

Reply:
Thanks. Way better than "fundamental" :)